transport_analysis/conductivity_msd.py

```diff
@@ -0,0 +1,474 @@
+                + (2 * self.cation_charge * self.anion_charge) * self._lij_cation_anion
+            )
+            fit_slope = np.gradient(
+                np.log(
+                    self._cond[self.linear_fit_window[0] : self.linear_fit_window[1]]
+                )[1:],
+                np.log(
+                    self._times[self.linear_fit_window[0] : self.linear_fit_window[1]]
+                    - self._times[0]
+                )[1:],
+            )
+            beta = np.nanmean(np.array(fit_slope))
+
+        else:
+            _, lij_cation_cation = np.polynomial.polynomial.polyfit(
+                self._times[1:],
+                self._msds_cation_cation[1:],
+                1,
+                w=self._lij_cation_cation_weights[1:],
+            )
+            _, lij_cation_anion = np.polynomial.polynomial.polyfit(
+                self._times[1:],
+                self._msds_cation_anion[1:],
+                1,
+                w=self._lij_cation_anion_weights[1:],
+            )
+            _, lij_anion_anion = np.polynomial.polynomial.polyfit(
+                self._times[1:],
+                self._msds_anion_anion[1:],
+                1,
+                w=self._lij_anion_anion_weights[1:],
+            )
+            cond = (
+                (self.cation_charge**2) * lij_cation_cation
+                + (self.anion_charge**2) * lij_anion_anion
+                + (2 * self.cation_charge * self.anion_charge) * lij_cation_anion
+            )
+
+            # fit_slope = np.gradient(np.log(self._cond)[1:], np.log(self._times - self._times[0])[1:])
+            # beta = np.average(np.array(fit_slope), weights=self.weights[1:])
+        if self.linear_fit_window:
+            self.results.linearity = beta
+        # self.results.fit_slope = cond
+        # self.results.fit_intercept = cond_intercept
+        cation_mass_fraction = (
+            self._cation_masses.sum() / self.allatomgroup.masses.sum()
+        )
+        anion_mass_fraction = self._anion_masses.sum() / self.allatomgroup.masses.sum()
+        solvent_fraction = 1 - cation_mass_fraction - anion_mass_fraction
+
+        self.results.conductivity = (
+            cond * (self.e_to_C * self.e_to_C) * (1 / (self.fs_to_s * self.A_to_m)) * 10
+        )
+        self.results.cation_transference_com = (
+            (self.cation_charge**2) * lij_cation_cation
+            + (self.cation_charge * self.anion_charge) * lij_cation_anion
+        ) / cond
+        self.results.anion_transference_com = (
+            (self.anion_charge**2) * lij_anion_anion
+            + (self.cation_charge * self.anion_charge) * lij_cation_anion
+        ) / cond
+        self.results.cation_transference_solvent = (
+            self.results.cation_transference_com - anion_mass_fraction
+        ) / solvent_fraction
+        self.results.anion_transference_solvent = (
+            self.results.anion_transference_com - cation_mass_fraction
+        ) / solvent_fraction
```

Comment:
The contents of results should be documented here. It's not clear to me what the difference between the `_solvent` and `_com` quantities is?